media-libs/openjpeg/openjpeg-2.5.0-r7.ebuild

```diff
@@ -0,0 +1,104 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake-multilib flag-o-matic
+
+# Make sure that test data are not newer than release;
+# otherwise we will see "Found-But-No-Test" test failures!
+#
+# To update: Go to https://github.com/uclouvain/openjpeg-data and grab the hash
+# of the latest possible commit whose commit date is older than the release
+# date.
+MY_TESTDATA_COMMIT="1f3d093030f9a0b43353ec6b48500f65786ff57a"
+
+DESCRIPTION="Open-source JPEG 2000 library"
+HOMEPAGE="https://www.openjpeg.org"
+SRC_URI="https://github.com/uclouvain/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	test? ( https://github.com/uclouvain/openjpeg-data/archive/${MY_TESTDATA_COMMIT}.tar.gz -> ${PN}-data_20210926.tar.gz )"
+
+LICENSE="BSD-2"
+SLOT="2/7" # based on SONAME
+KEYWORDS="~alpha amd64 arm arm64 hppa ~ia64 ~loong ~m68k ~mips ppc ppc64 ~riscv ~s390 sparc x86 ~amd64-linux ~x86-linux ~arm64-macos ~ppc-macos ~x64-macos ~x64-solaris"
```

Comment:
We shouldn't do it straight-to-stable. It will be shadowed in stable though, yeah, so we may as well file a stablereq immediately once it's merged.

Reply:
Ah, you're right, thanks for taking a look. Maybe we can just drop this PR in favour and just wait for 2.5.2 to get stabilized? It has the same security fix.

Reply:
Now that we're past 30 days with 2.5.2, I've gone ahead and [filed a stablereq](https://bugs.gentoo.org/930395). Not sure there's a necessity to backport at this point.

Reply:
Thanks, I agree. Closing.